### PR TITLE
202203 report

### PR DESCRIPTION
**Pull request by brownsarahm**
Apologies for so much lateness, end of the semester was hard. 

I don't know how but the notes about future plans I took in the March meeting were not in the file when I opened it back up. Please recommend additions.

### CODE REVIEW

File: reports/2022-03-report-to-ec.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## Summary 
+
+In our first term as trainers leadership, we worked mostly on organizational matters.  We set up the infrastructure for the group to be successful and transparent in their operations going forward. We also worked carefully to flesh out details of how operations would balance agency with consensus.  Our biggest project completed beyond internal documentation was the transition from the Trainer Agreement to the new Trainer Certification Renewal Process. 
+
+
+
+<!-- #region -->
+## Certification Renewal
+
+With the community, we agreed to and got approved a replacement to the trainer agreement.  The new certificaiton renewal makes it clear that the participation requirement is a requirement to remain a badged trainer, not a commitment of future service.  This took a lot of collaboration with the DIT and revision and substantial trainer community feedback to achieve. After approving the general Renewal process for the long term, we approved the 2022 specific Implementation to provide a clear and transparent community governed process that the Core team leads.
```
Comment (jcoliver) [inline reports/2022-03-report-to-ec.md:13]: (letting my pedantic flag fly!) replace "DIT" with "Director of Instructor Training"

Comment (brownsarahm) [inline reports/2022-03-report-to-ec.md:13]:
```suggestion
With the community, we agreed to and got approved a replacement to the trainer agreement.  The new certification renewal makes it clear that the participation requirement is a requirement to remain a badged trainer, not a commitment of future service.  This took a lot of collaboration with the Director of Instructor Training and revision and substantial trainer community feedback to achieve. After approving the general Renewal process for the long term, we approved the 2022 specific Implementation to provide a clear and transparent community governed process that the Core team leads. 
```

Comment (brownsarahm) [reply reports/2022-03-report-to-ec.md]: thanks!

**Review by marklcrowe — APPROVED** [review]

**Review by paocorrales — APPROVED** [review]
Everything looks great! Although, I feel that I don't have much to say about this a I'm new to the role :)

**Review by jcoliver — APPROVED** [review]

**Comment by karenword** [comment]
Maybe in the forward-looking section it would be appropriate to say something about creating approachable and appealing opportunities for Trainers to engage with the community / work toward community sustainability?

**Review by bonnyad — APPROVED** [review]

**Pull request merged**